src/main/java/bio/terra/service/dataset/flight/create/CreateDatasetMetadataStep.java

```diff
@@ -49,9 +48,9 @@ public StepResult doStep(FlightContext context) {
         } catch (DuplicateKeyException dkEx) {
             return new StepResult(StepStatus.STEP_RESULT_FAILURE_FATAL,
                 new InvalidDatasetException("Dataset name already exists: " + datasetRequest.getName(), dkEx));
-        } catch (SQLException sqlEx) {
+        } catch (Exception ex) {
```

Comment:
Was this change also just for testing? You could `catch` "other errors" separately, but this change seems reasonable to me either way.

Reply:
The existing exception message wasn't SQL-specific, so I thought it'd be reasonable to widen the `catch` here. I could go the opposite direction and catch each type individually, with tailored messages in the rethrown exceptions, if it'd be useful.